misc/python/materialize/output_consistency/query/query_result.py

```diff
@@ -32,6 +35,11 @@ def __init__(self, query_template: QueryTemplate, query_id: str):
         self.outcomes: List[QueryOutcome] = []
         self.query_template = query_template
 
+    def get_outcome_by_strategy_key(self) -> Dict[EvaluationStrategyKey, QueryOutcome]:
+        return dict(
+            [(outcome.strategy.identifier, outcome) for outcome in self.outcomes]
+        )
```

Comment:
```suggestion
        return  {outcome.strategy.identifier: outcome for outcome in self.outcomes}
```
My brain is slow today. No need to go list -> dict, but doesn't matter much.